tutorials/inputs/mouse_and_input_coordinates.rst

```diff
@@ -62,3 +62,7 @@ Alternatively, it's possible to ask the viewport for the mouse position:
  .. code-tab:: csharp
 
     GetViewport().GetMousePosition();
+Additional notes
```

Comment:
Please leave a blank line after the code use the `.. note::` annotation to create a note block, instead of adding a heading.

```
.. note:: When the mouse mode is set to ``Input.MOUSE_MODE_CAPTURED``, the ``event.position`` value from ``InputEventMouseMotion`` is the center of the screen.
          Use ``event.relative`` instead.
```